personas.md

```diff
@@ -0,0 +1,38 @@
+# OSAC Personas
+
+These are the essential personas that OSAC must target for an MVP. Many
+additional personas can be identified within the solution space, some of which
+will be added to this document over time as the project is ready to address
+their unique use cases.
+
+## Cloud Provider Admin
```

Comment:
This persona is sometimes broken down into sub-personas, such as network admin, storage admin, security admin, and tenant onboarder, among others.

Reply:
done

Reply:
I figure this whole project is a solution for cloud providers, so the audience reading this is most likely to be a cloud provider admin or similar who wants to offer these services. So leading with their use cases would be most relatable and let the reader get oriented.